.travis.yml

```diff
@@ -3,19 +3,19 @@ sudo: false
 matrix:
   include:
     - scala: 2.10.6
-      # script: ./sbt ++$TRAVIS_SCALA_VERSION clean test mimaReportBinaryIssues"
-      script: ./sbt ++$TRAVIS_SCALA_VERSION clean test
+      script: ./sbt ++$TRAVIS_SCALA_VERSION clean test mimaReportBinaryIssues
+      #script: ./sbt ++$TRAVIS_SCALA_VERSION clean test
```

Comment:
I left them to make it easier in the future since we will, probably, turn it off again.